taf/updater/updater.py

```diff
@@ -625,6 +639,7 @@ def validate_repository(
     validate_from_commit=None,
     excluded_target_globs=None,
     strict=False,
+    bare=False,
```

Comment:
Validation is only run when the repositories already exist, to validate local repositories. So, in this case, we could check if the repositories are bare.

Reply:
Are you talking about adding bare as a parameter to the validate_repository function? If so I'd have taf repo validate check if the auth_repo is a bare repo.